iele-gas.md

```diff
@@ -576,19 +591,18 @@ constant cost to perform the jump and store the return address.
       requires notBool IDX in_keys(LABELS)
 ```
 
-The cost to return from an intra-contract call is the cost to move the return values into the result registers plus the cost to
-jump to the return address.
+The cost to return is folded into the cost of the call instruction.
```

Comment:
This comment looks like it is only correct when `Gnewmove << SCHED >>` is true.

Reply:
Since Gnewmove is always true in new versions of IELE, I think that's fine.